src/EnergyPlus/WaterThermalTanks.hh

```diff
@@ -229,10 +229,10 @@ namespace WaterThermalTanks {
         // Members
         std::string Name;                                   // Name of heat pump water heater
         std::string Type;                                   // Type of water heater (HEAT PUMP:WATER HEATER)
-        int TypeNum;                                        // integer parameter for heat pump water heater
+        DataPlant::PlantEquipmentType TypeNum;              // integer parameter for heat pump water heater
         std::string TankName;                               // Name of tank associated with heat pump water heater
         std::string TankType;                               // Type of water heater (MIXED or STRATIFIED) used with heat pump
-        int TankTypeNum;                                    // Parameter for tank type (MIXED or STRATIFIED)
+        DataPlant::PlantEquipmentType TankTypeNum;          // Parameter for tank type (MIXED or STRATIFIED)
```

Comment:
Can TypeNum and TankTypeNum be combined?

Reply:
If there is only a single global `enum class PlantEquipmentType` then all plant components should use it and there should not be component sub-class specific `enums`. Get rid of `Num` here while you are at it. 😁

Reply:
Will do!